preparing-to-code/readme.md

```diff
@@ -10,3 +10,10 @@ In these lessons you will learn how to navigate MaxOSX, learn about the language
 | 05 | [Ada programming languages](./ada-languages) |
 | 06 | [Terminal](./terminal/) |
 | 07 | [Environment Setup](./environment-setup/) |
+
+Notes:
+* If you are *a student enrolled to begin your cohort in upcoming months*, you are required to follow all the lessons.
+* If you are *an applicant and do not currently have a machine with Mac OS X*, please complete the lessons on [Effective Internet Searching](./internet-searching/) and [Ada programming languages](./ada-languages). On your Windows machine, to get your *environment setup*, please use the following links and follow instructions to install the these applications.
+- [Google Chrome](https://www.google.com/chrome/)
+- [Atom]( https://atom.io/)
+-[Ruby](https://www.ruby-lang.org/en/documentation/installation/#rubyinstaller) Install the latest stable released version of Ruby (not a preview release). Once Ruby is installed, Interactive Ruby (`irb`) should be available on the command prompt on your Windows machine. Read more on [irb](https://github.com/Ada-Developers-Academy/jump-start/tree/master/preparing-to-code/environment-setup#interactive-ruby)
```

Comment:
the formatting here is off - i think you might just need an additional space

Reply:
Good catch! I had missed a space between `-` and `[Ruby]`. Fixing that did the trick.
Note to self, use the markdown preview more often.